packages/flutter/lib/src/widgets/debug.dart

```diff
@@ -132,6 +132,17 @@ bool debugProfileBuildsEnabled = false;
 ///    for every widget and has a higher overhead cost.
 bool debugProfileBuildsEnabledUserWidgets = false;
 
+/// Adds debugging information to the args of [Timeline] events related to
+/// [Widget] builds.
+///
+/// This flag will only add [Timeline] arguments for debug builds and may
```

Comment:
Can we improve the docs here a little bit and describe what additional information one can expect with these flags enabled and what the "other conditions" are? Right now, just from reading this doc I have no idea when I would ever turn this flag on...

Reply:
improved the docs. ptal.